ding_readers_compass.module

```diff
@@ -16,12 +18,3 @@ function ding_readers_compass_ctools_plugin_directory($module, $plugin) {
     return "plugins/$plugin";
   }
 }
-
```

Comment:
Why is this removed?

Reply:
Because this is automatically exported by Features module into ```ding_readers_compass.features.inc``` file.

Reply:
I dont think we should manipulate this either